lisp/ess-mode.el

```diff
@@ -107,7 +107,8 @@
     (define-key map "\C-c\C-e"   'ess-extra-map)
     (define-key map "\C-c\C-t"   'ess-dev-map)
     (when ess-smart-S-assign-key
-      (define-key map ess-smart-S-assign-key #'ess-smart-S-assign))
+      (define-key map ess-smart-S-assign-key 'ess-insert-S-assign))
+    (define-key map (kbd "C-c C-=") 'ess-cycle-assignment)
```

Comment:
BTW, I am divided if we should use `C-M-=` for this. I don't think that it's a violation of emacs convention as it's generally regarded ok to rebind standard sequences:

> It is also legitimate for a major mode to rebind a standard key sequence whose standard meaning is rarely useful in that mode.

And C-M-= is not even a standard sequence. Thoughts?

Reply:
I think it's OK to bind `C-M-=`, but I personally find `C-c C-=` easier to type. I don't have a strong opinion though.

Reply:
Nevermind. I thought C-M-= is much easier to type, but it looks that it's "just me".